### PR TITLE
i have some problem with demo link early.  

### PR DESCRIPTION
**Pull request by Kateryna199420**
- [DEMO LINK](https://Kateryna199420.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Kateryna199420.github.io/layout_search-bar-airbnb/report/html_report/)        
- - [x ] Icon implemented using background-image CSS property
- [ x] Inputs are written inside of 'form' tag with correctly passed attributes
- [ x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x ] Code follows all the [Code Style Rules ❗️](./checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,25 @@
...
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="searchbar__form" data-qa="big">
+      <label class="searchbar__label searchbar__label--big">
+        <input
+        type="text"
+        class="searchbar__input searchbar__input--big"
+        placeholder="Try “Los Angeles”"
+        data-qa="keypress"
+        >
```
Comment (OleziO) [inline src/index.html:21]:
```suggestion
        <input
          type="text"
          class="searchbar__input searchbar__input--big"
          placeholder="Try “Los Angeles”"
          data-qa="keypress"
        >
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,25 @@
...
+      </label>
+    </form>
+
+    <form class="searchbar__form" data-qa="small">
+      <label class="searchbar__label searchbar__label--small">
+        <input
+        type="text"
+        class="searchbar__input searchbar__input--small"
+        placeholder="Try “Los Angeles”"
+        >
```
Comment (OleziO) [inline src/index.html:31]:
```suggestion
        <input
          type="text"
          class="searchbar__input searchbar__input--small"
          placeholder="Try “Los Angeles”"
        >
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,84 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+
+  src: url(./fonts/Avenir-Book.ttf);
+
+  font-style: normal;
+
+  font-weight: 300;
```
Comment (OleziO) [inline src/style.css:8]:
```suggestion
  font-family: Avenir;
  src: url(./fonts/Avenir-Book.ttf);
  font-style: normal;
  font-weight: 300;
```

File: src/style.css [hunk 4]
```diff
@@ -1 +1,84 @@
...
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  --text-color: #3D4E61;
+  --border-color: #E1E7ED;
```
Comment (OleziO) [inline src/style.css:17]: Move all variables initialization to the :root selector.

**Review by OleziO — CHANGES_REQUESTED** [review]

**Review by VitaliyBondarenko1982 — APPROVED** [review]
GJ!

**Pull request closed without merging**